api/src/paths/project/{projectId}/survey/{surveyId}/sample-site/{surveySampleSiteId}/index.ts

```diff
@@ -165,7 +165,6 @@ PUT.apiDoc = {
                   type: 'array',
                   items: {
                     type: 'object',
-                    additionalProperties: false,
```

Comment:
additionalProperties: false is included back in the openAPI spec in a later PR. I removed it in this PR to allow for sampling sites to be edited, to properly test this PR.